src/pspm_convert_ppg2hb.m

```diff
@@ -13,16 +13,16 @@
 %            channel: ppg channel number, default: last ppg channel
```

Comment:
The argument `channel` should be removed from the help.

Reply:
Yes... I have removed this line now. Thanks.